Cargo.toml

```diff
@@ -15,19 +15,27 @@ panic = "abort"
 
 [workspace.dependencies]
 anyhow = { version = "1", default-features = false }
-camino = { version = "1.1.7" }
-dice-verifier = { git = "https://github.com/oxidecomputer/dice-util", rev = "d332e43d2269cc6a566892386be63fe6adf3052f" }
+camino = { version = "1.1.7", default-features = false, features = ["serde1"] }
+attest-data = { git = "https://github.com/oxidecomputer/dice-util", branch = "attest_messages_tq"}
+clap = { version = "4", default-features = false, features = ["std", "derive", "default", "wrap_help"] }
+ciborium = "0.2.2"
+cfg-if = "1.0"
+dice-verifier = { git = "https://github.com/oxidecomputer/dice-util", branch = "attest_messages_tq"}
```

Comment:
I personally think it's safer to use a hash here. Please update to the latest before merge.